File: Web/app/model/Tambol.js
Ext.define(AppConfig.appName + '.model.Tambol', {
    extend: 'Ext.data.Model',
    xtype: 'tambol',
    idProperty: 'Seq',
    fields: [
        { name: 'Seq' },
        { name: 'ProvinceCode', type: 'string' },
        { name: 'AmphurCode', type: 'string' },
        { name: 'TamCode', type: 'string' },
        { name: 'TamName', type: 'string' }
    ]
});

File: Cwn.Doe.BusinessModels/Entities/Tambol.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Cwn.Doe.BusinessModels.Entities
{
    public class Tambol
    {
        public virtual long Seq { get; protected set; }

        public virtual string ProvinceCode { get; set; }
        public virtual string AmphurCode { get; set; }

        public virtual string TamCode { get; set; }
        public virtual string TamName { get; set; }
    }
}
